duckduckgo/BrowserServicesKit
# Fingerprint privacy reference tests

Task/Issue URL:  https://app.asana.com/0/0/1203391706352066/f
iOS PR:  N/A
macOS PR: N/A
What kind of version bump will this require?: N/A (only tests were added)

**Description**:
Add privacy reference tests for fingerprinting protection

**Steps to test this PR**:
1. Make sure https://github.com/duckduckgo/privacy-reference-tests/pull/66 is merged
1. Run tests 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
